### ISSUE TITLE
support from oxid developers

### ISSUE THREAD

**Issue by keywan-ghadami-oxid** [acirtautas/oxid-module-internals#17]
Hi Alfonsas Cirtautas, i find you tool very very useful for the oxid community so i like to invite more oxid developers to maintain it. Would you like to give me (and maybe other oxid developers) write access, or do you like me to fork?

**Comment by suabo** [comment]
Why not merge it into the next ce release? This is a must have module for OXID. There is no other way to update Modules without deaktivating them without this module, isnt it? The Reson why there is no way, when this module can handle it is another good Question i think. Please merge it to the next OXID release! Thx

**Comment by keywan-ghadami-oxid** [comment]
what about license? It is open source and oxid ce is base for oxid ee and oxid pe and so would need an something like an contributor agreement. But anyway it was already decided in past to not merge it, i guess more people have to leave comments like yours.

**Comment by netensio** [comment]
A pity that it won't be merged to core functionality.

**Comment by marsium** [comment]
Ich finde die Feature sehr nutzlich und schließe mich den obigen Kommentaren ein.

**Comment by akdes** [comment]
Erstes Modul, welches installiert wird. Ohne geht es bei uns gar nicht mehr. Bitte ins Package aufnehmen.

**Comment by keywan-ghadami-oxid** [comment]
until we get this module/or at least some of the functionality somehow bundled we(volunteers) are going to maintain in a fork https://github.com/OXIDprojects/oxid-module-internals
Please ask me if you need write access, or if you need support
EDIT 12/2018 (update github url)

**Comment by acirtautas** [comment]
Hi Guys, sorry for not responding so long.

I created this module purely out of the need and thought it could help other oxid developers.

I would be happy to give write access to more developers.

**Comment by keywan-ghadami-oxid** [comment]
Hi i started to maintain a fork of this module in a community space (https://github.com/OXIDprojects/oxid-module-internals), where everybody can contribute directly.
If you like you like to, you can transfer this repository to my account so i will take care that it will become the main project (see settings of this project in github to transfer ownership), or you contact me directly if you like.
One major problem i have now is the missing license of this module see https://github.com/acirtautas/oxid-module-internals/issues/18 please give a statement under which license we can use the code.  
EDIT: 12/2018 (update github url)

**Comment by acirtautas** [comment]
Hi @keywan-ghadami-oxid with help from @kermie I have finally added the license.

**Comment by suabo** [comment]
So Maybe it can be included by default into OXID 6.2 or something?

**Comment by kermie** [comment]
It doesn't make much sense, @suabo: this module is mostly used by developers when setting up a project and other modules will be installed. Later, for the online merchant, it is not useful to change the module setup nor to use this module which checks it.

**Comment by smxsm** [comment]
I strongly disagree, @kermie :) we use it especially in live shops, if we try to analyze strange behaviour in the shop... and a quick look into "module internals" shows that some class extension was deactivated at the last deployment, or a new template block is missing, or somebody forgot to deploy a new controller etc. - so it is helpful for development and live checks and should indeed be standard functionality for the shop imho ... without it you are pretty lost if something is wrong with any module.

**Comment by jkrug** [comment]
Well i think the module overall is a bit too huge. But in general having trouble in a live sstem with modules is a hell on debugging beause there are no informations about a lot of settings (like "files" entries in the metadata,...

**Comment by acirtautas** [comment]
Hi everyone, it is a pleasure to hear that this tiny module is useful. I would agree to @kermie on keeping it as a module.
If few of you want to join module maintenance I would happily give you all necessary permissions :)

**Comment by kermie** [comment]
Thanks @acirtautas, hope you're well, mate! We are already a bit ahead in this fork: https://github.com/acirtautas/oxid-module-internals/compare/master...OXIDprojects:master, don't know anything about the others. What is the best way to handle it - maybe I send a pull request from OXIDforge <- acirtautas, you merge it, I kill the OXIDforge fork and you hand the repository over to OXIDforge?
